### PR TITLE
DEV-3182-new-go-exercise-fishandchips

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: subjects/fishandchips/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Fishandchips
```
Comment (Hamzaelkhatri) [inline subjects/fishandchips/README.md:1]: The title should be in lower case `fishandchips`

File: subjects/fishandchips/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Instructions
+
+Write a function called `Fishandchips()` Write a function that takes an integer and returns a string.
```
Comment (Hamzaelkhatri) [inline subjects/fishandchips/README.md:5]: there is `Write a function` two time at the line

**Review by Hamzaelkhatri — DISMISSED** [review]

File: subjects/fishandchips/README.md (hunk 1 above)
Comment (zainabdnaya) [reply subjects/fishandchips/README.md]: it's fixed

File: subjects/fishandchips/README.md (hunk 2 above)
Comment (zainabdnaya) [reply subjects/fishandchips/README.md]: it's fixed

**Review by jotapero — APPROVED** [review]
Looks good

**Pull request merged**